awsconfigs/common/istio-ingress/overlays/api/params.env

```diff
@@ -0,0 +1,6 @@
+# ARN of an ACM certificate to configure in ALB's HTTPS listener rule
+certArn=
+# HttpHeaderName for the HTTP header condition
+httpHeaderName=x-api-key
+# Values for the HTTP header condition
+httpHeaderValues=["token1", "token2"]
```

Comment:
what are these token used for ? needed by kserve?

Reply:
I think you answered it later in the PR, its used by ALB for rule validation. added few more lines here to clarify